vulkan_dump_resources.md

```diff
@@ -315,72 +316,94 @@ Image descriptors can be fine grained further by specifying the desired subresou
-              is generating one output json file that contains the information for all dumped commands.
-  --dump-resources-dump-immutable-resources
-              Enables dumping of resources that are used as inputs in the commands requested for dumping
-  --dump-resources-dump-all-image-subresources
-              Enables dumping of all image sub resources (mip map levels and array layers)
-  --dump-resources-dump-raw-images
-              When enabled all image resources will be dumped verbatim as raw bin files.
-  --dump-resources-dump-separate-alpha
-              When enabled alpha channel of dumped images will be dumped in a separate file.
-  --dump-resources-binary-file-compression-type
-              Compress files that are dumped as binary. Available compression types are: [none, lz4 (block format), zlib,
-              zlib, zstd]. Default is none (no compression).
-  --dump-resources-dump-build-acceleration-structures-input-buffers
-              Dump all input buffers used in vkCmdBuildAccelerationStructures. This includes vertex, index,
-              transformation matrix, AABB and instance buffers. Default is off.
+## Vulkan dump resources options
+
+### gfxrecon-replay command line parameters
+
+Dump resources feature can be controled in several ways. This is done either by providing the appropriate options via the command line or using the input json.
+The majority of the options are only available via the input json. The command line options, which can be passed to either to the `gfxrecon-replay` tool or to the Android application through the `gfxrecon.py` script are the following:
+
+- ```--dump-resources <filename>.json```
+Specify the input json file. It should contain the command indices and the various options that control how and which the resources should be dumped. The format for the json file is documented in detail in the gfxreconstruct documentation.
+
+- ```--dump-resources-dir <dir>```
+Directory to write dump resources output files. Default is the current working directory.
+
+
+### Input json options
+
+The rest of the dump resources options are available through the input json file. They are expected to be inside a json object named `DumpResourcesOptions`.
+The possible options are the following:
+
+* `Scale` <scale>: Scale images generated by dump resources by the given scale factor. The scale factor must be a floating point number greater than 0. Values greater than 10 are capped at 10. Default value is 1.0.
+
+* `ImageFormat` <format>:
+Image file format to use for image resource dumping. Available formats are: `bmp` and `png`. Default is `bmp`.
+
+* `DumpBeforeCommand` <`true` | `false`>:
+In addition to dumping gpu resources after the CmdDraw, CmdDispatch and CmdTraceRays calls also dump resources before those calls.
+
+* `DumpDepth`
+Configures whether to dump the depth attachment when dumping draw calls. Default is disabled.
```

Comment:
Change to: "Enables dumping of the depth attachment when dumping draw calls. Default is disabled."

Reply:
fixed